libs/components/lookup/testing/src/search/search-harness.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('Search harness', () => {
     const fixture = TestBed.createComponent(SearchHarnessTestComponent);
     const loader = TestbedHarnessEnvironment.loader(fixture);
 
-    let searchHarness: SkySearchHarness;
+    let searchHarness: SkySearchHarness | undefined;
```

Comment:
I don't see any calls to this function that don't supply `options.dataSkyId`, so we can remove the `| undefined` and the check for a falsy `options.dataSkyId`. Then you can get rid of all the Elvis operators.